manifests/config.pp

```diff
@@ -178,6 +178,18 @@
       }
     }
     'systemd': {
+      # Puppet 5 doesn't have https://tickets.puppetlabs.com/browse/PUP-3483
```

Comment:
please change this line to:
```puppet
# in Puppet 5 the systemd provider doesn't scan for unit file changes (see ticket URL)

Reply:
@igalic This is just being moved from somewhere else.

Reply:
Pretty sure that wording was mine. https://github.com/voxpupuli/puppet-prometheus/commit/8588d880a6c0736e95d90977aa9f4465239961ae#diff-0ac866172efd29d0ddc2aa75ae6f182dR188